commands/slash/General/serverinfo.js

```diff
@@ -0,0 +1,44 @@
+﻿const { EmbedBuilder } = require('discord.js');
+
+module.exports = {
+    name: "serverinfo", // Name of command
+    description: "gets detailed info for a server", // Command description
+    type: 1, // Command type
+    options: [{
+        name: "server",
```

Comment:
If the select menu from /execute is preferred, it could be added to this command, as well, instead of a server id.